65.md

```diff
@@ -6,59 +6,138 @@ Relay List Metadata
 
 `draft` `optional`
 
-Defines a replaceable event using `kind:10002` to advertise preferred relays for discovering a user's content and receiving fresh content from others.
+Defines two replaceable events to advertise preferred relays for discovering a user's content and receiving fresh content from others.
 
-The event MUST include a list of `r` tags with relay URIs and a `read` or `write` marker. Relays marked as `read` / `write` are called READ / WRITE relays, respectively. If the marker is omitted, the relay is used for both purposes.
+Relay Usages
+------------
+
+There are three defined usages for a relay:
+
+- `INBOX` relays are for events that the relay list owner will be reading, specifically watching for events that tag them. These do not include DMs (except in the deprecated form of relay list).
+
+- `OUTBOX` relays are for events that the relay list owner has authored and is publishing for public consumption.
+
+- `PRIVATE INBOX` relays are like INBOX relays, except they are for reciept of private messages such as perhaps direct messages (DMs), giftwraps, or the like (but this NIP does not specify what kinds should use relays with this designator).
+
+### When to use which
+
+When posting events (except private ones):
+
+- Clients SHOULD post events to all of their client user's OUTBOX relays,
+- For each p-tag in a posted event, clients SHOULD post the event to at least some of the INBOX relays of the tagged person.
+
+When posting private messages:
+
+- Clients SHOULD post events to at least some (preferably all) of the the p-tag recipient's PRIVATE INBOX relays.
+- Clients SHOULD post a copy to their client user's own PRIVATE INBOX relays.
+- If a user has no PRIVATE INBOX relays, INBOX relays should be used instead.
```

Comment:
Better wording: 

When publishing public events:

- Clients SHOULD publish events to all of the signer's OUTBOX relays,
- Clients SHOULD publish the event to at least some of the INBOX relays of each tagged key.

When sending private messages:

- Clients SHOULD publish events to at least some (preferably all) PRIVATE INBOX relays of each of the recipients.
- Clients SHOULD publish a copy to their client user's own PRIVATE INBOX relays.

Reply:
I've pushed changes very close to your suggestion.